editors/vscode/syntaxes/lp.tmLanguage.json

```diff
@@ -44,7 +40,7 @@
     },
 
     "tactics": {
-      "match": "\\b(apply|assume|fail|focus|generalize|have|induction|orelse|refine|reflexivity|remove|repeat|rewrite|set|simplify|solve|symmetry|try|why3)\\b",
+      "match": "\\b(apply|assume|focus|generalize|have|induction|orelse|refine|reflexivity|remove|repeat|rewrite|set|simplify|solve|symmetry|try|why3)\\b",
```

Comment:
fail is a tactic (https://github.com/Deducteam/lambdapi/blob/00c63619580c2bf679ec6ee85ba5161da8cd2cb7/src/handle/tactic.ml#L458); we should keep it here I guess